Vagrantfile

```diff
@@ -7,6 +7,7 @@ VAGRANTFILE_API_VERSION = '2'
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box = 'ubuntu/xenial64'
   config.vm.hostname = "#{HOSTNAME}"
+  config.vm.network "forwarded_port", guest: 22, host: rand(30000)+1234
```

Comment:
```
$ vagrant --version
Vagrant 1.8.6
```
Sure, I will try the official way: `auto_correct: true` from: https://www.vagrantup.com/docs/networking/forwarded_ports.html